app/lib/widgets/ToDoListView.dart

```diff
@@ -25,6 +28,7 @@ class _ToDoListViewState extends State<ToDoListView> {
   ToDoController todoController = ToDoController.instance;
   late List<ToDoTaskItem> completedTasks;
   late List<ToDoTaskItem> pendingTasks;
+  bool isCardExpanded = false;
 
   @override
   void initState() {
```

Comment:
We have ```toggleCheck()```helper function inside ```todo_controller.dart``` which is handling the state of tasks. Also ```setState(){}``` shouldn't be used directly inside ```initState()``` method. Either use ```if (mounted){}``` wrap before calling ```setState(){}``` or use GetX state option and helper function that already exists which is better to keep state management pattern.